src/ui/database/common_event_widget.cpp

```diff
@@ -43,9 +48,18 @@ void CommonEventWidget::setData(lcf::rpg::CommonEvent* common_event) {
 	LcfWidgetBinding::bind(ui->lineName, m_current->name);
 	LcfWidgetBinding::bind(ui->comboTrigger, m_current->trigger);
 	LcfWidgetBinding::bind(ui->groupSwitch, m_current->switch_flag);
-	LcfWidgetBinding::bind(ui->comboSwitch->comboBox(), m_current->switch_id);
+	LcfWidgetBinding::bind(ui->comboSwitch, m_current->switch_id);
 
 	ui->commands->setData(m_project, m_current);
 
 	this->setEnabled(common_event != nullptr);
+	updateComboSwitchEnabled();
+}
+
+void CommonEventWidget::on_comboTrigger_currentIndexChanged(int index) {
+	updateComboSwitchEnabled();
+}
+
+void CommonEventWidget::updateComboSwitchEnabled() {
+	ui->groupSwitch->setEnabled(ui->comboTrigger->currentData().toInt() != 5);
```

Comment:
also here with the 5 constant

Reply:
Fixed (using `lcf::rpg::CommonEvent::Trigger_call` now).